src/components/Firestore/index.tsx

```diff
@@ -94,14 +96,32 @@ export const Firestore: React.FC<FirestoreProps> = ({ config, projectId }) => {
   async function handleClearData(api: DatabaseApi) {
     const shouldNuke = await promptClearAll();
     if (!shouldNuke) return;
-    api.nukeDocuments();
+    setIsRefreshing(true);
+    api.nukeDocuments().then(() => {
```

Comment:
no await inside async func? is that intentional?

Reply:
Ah of course, added.